diff --git a/min.nimble b/min.nimble
index 52f861e..6a4bcc2 100644
--- a/min.nimble
+++ b/min.nimble
@@ -3,11 +3,11 @@ import
 
 # Package
 
-version       = pkgVersion
-author        = pkgAuthor
-description   = pkgDescription
+version       = "0.39.0"
+author        = "h3rald"
+description   = ""
 license       = "MIT"
-bin           = @[pkgName]
+bin           = @["min"]
 installExt    = @["nim", "c", "h", "a"]
 installFiles  = @["min.yml", "min.nim", "prelude.min", "help.json"]
 installDirs   = @["minpkg"]
@@ -15,8 +15,3 @@ installDirs   = @["minpkg"]
 # Dependencies
 
 requires "nim >= 1.6.2, zippy >= 0.5.6"
-
-before install:
-  exec "nimble install -y nifty"
-  exec "nifty remove -f"
-  exec "nifty install"
